### PR TITLE
[INT-782] Set sideMenuLinks false

### PR DESCRIPTION
**Pull request by jorgelcff**

### CODE REVIEW

File: src/components/HamburgerButton/index.tsx [hunk 1]
```diff
@@ -1,14 +1,24 @@
...
   isOpen = false,
   onClick = () => {}
 }: HamburgerButtonProps) => {
+  if (links.length === 0){
+    return(
+      <>
+      </>
+    )
+  }
+
```
Comment (dcruzb) [inline src/components/HamburgerButton/index.tsx:21]: The button shouldn't be aware of links

File: src/lib-components/Navbar/index.tsx [hunk 2]
```diff
@@ -216,6 +215,7 @@ export const Navbar = ({
                 <HamburgerButton
                   isOpen={sideMenuIsOpen}
                   onClick={handleToggleSideMenu}
+                  links={sideMenuLinks}
```
Comment (dcruzb) [inline src/lib-components/Navbar/index.tsx:218]: The button shouldn't be aware of links, rather it should not exist if there are no links

**Review by dcruzb — CHANGES_REQUESTED** [review]

**Review by jorgelcff — COMMENTED** [review]
Mudança na forma de exibir o menu lateral.

**Review by jorgelcff — COMMENTED** [review]
Chave handleToggleSideMenu fechada

**Review by dcruzb — APPROVED** [review]

**Pull request merged**